app/controllers/episodes_controller.rb

```diff
@@ -1,12 +1,14 @@
 class EpisodesController < ApplicationController
   load_and_authorize_resource
-  before_action :set_episode, only: %i[edit show update]
+  before_action :set_episode, except: %i[index create episode_params new]
 
   def index
     @episodes = Episode.order(created_at: :desc).page params[:page]
   end
 
-  def show; end
+  def show
```

Comment:
Why we need to update something on show?

Reply:
I moved it to set_episode. You see, this method will be strongly connected with podcast statuses which are not merged to master yet. So I just left it there for testing.